src/test/java/org/apache/sling/resourceresolver/impl/mapping/ResourceMapperImplTest.java

```diff
@@ -355,6 +358,31 @@ public void mapNestedAliasTarget() {
                 .allMappingsWithRequest("/app/alias-parent/alias-child", "/app/alias-parent/child", "/app/parent/alias-child")
                 .verify(resolver, req);
     }
+    
+    /**
+     * Validates that mappings for an empty path
+     *
+     * @throws LoginException
+     */
+    @Test
+    public void mapEmptyPathWithNonExistingResource() {
```

Comment:
```suggestion
    public void mapEmptyPathWithUnreadableRoot() {
````

Reply:
Done

Reply:
@humbledev123 where is this done? I don't see any new commits added to this PR.